test/evergreen.yml

```diff
@@ -558,6 +558,15 @@ variables:
         vars:
           format_test_script_args: -e "SEGFAULT_SIGNALS=all" -b "catchsegv ./t" -t 360
 
+  - &format-abort-stress-test
```

Comment:
Can we rename this task to `format-abort-recovery-stress-test` ?

Reply:
Yes, that's a better name to reflect what the test does.

Reply:
8b200e3f990b583363e458f1f14a0420172f60d0